internal/abi/abi_amd64.h

```diff
@@ -0,0 +1,99 @@
+// Copyright 2021 The Go Authors. All rights reserved.
```

Comment:
Yes it is copied from runtime/cgo. What should be included in this README?

Reply:
What files are copied from the Go runtime.

On second thought, with `zcallback_darwin_*.s`, we can add the note to the exisitng README instead of adding a new README in this directory

Reply:
So add a note about both zcallback and internal/abi? What should it include?

Reply:
Please list all the files from the Go runtime and add a note that the license is BSD2 for these files.

Reply:
Does the README.md look good?

Reply:
Yes!